src/gen/js/genOperations.ts

```diff
@@ -238,7 +238,24 @@ function groupParams(groups: any, param: ApiOperationParam): any {
   const group = groups[param.in] || []
   const name = getParamName(param.name)
   const realName = /^[_$a-z0-9]+$/gim.test(param.name) ? param.name : `'${param.name}'`
-  const value = param.required ? name : 'options.' + name
+  const value = param.required ? name : 'options.' + name + '||' + [param.type].map(v => {
+      switch(v) {
+        case 'file':
+          return 'undefined'
+        case 'string':
+          return "''"
+        case 'number':
+          return '-1'
+        case 'boolean':
+          return 'false'
+        case 'integer':
+          return '-1'
+        case 'array':
+          return '[]'
+        default:
+          return 'undefined'
+      }
```

Comment:
Looking at this it appears to be assigning defaults to empty optional params, which may rightly have no value. This would unfortunately also be breaking (also looks like it would swap `0` for `-1`).

`undefined` for all of these is a valid default. If a param should have a true default then that should be defined in the open api spec.

Let me know if i'm overlooking something.